src/node_env_var.cc

```diff
@@ -353,6 +354,15 @@ MaybeLocal<Object> CreateEnvVarProxy(Local<Context> context,
   env_proxy_template->SetHandler(NamedPropertyHandlerConfiguration(
       EnvGetter, EnvSetter, EnvQuery, EnvDeleter, EnvEnumerator, data,
       PropertyHandlerFlags::kHasNoSideEffect));
-  return scope.EscapeMaybe(env_proxy_template->NewInstance(context));
+
+  MaybeLocal<Object> env_proxy_ml = env_proxy_template->NewInstance(context);
+  Local<Object> env_proxy;
+  if (!env_proxy_ml.ToLocal(&env_proxy)) {
+    return MaybeLocal<Object>();
+  }
+
+  CHECK(env_proxy->SetPrototype(context, Null(isolate)).FromJust());
```

Comment:
```suggestion
  if (env_proxy->SetPrototype(context, Null(isolate)).IsNothing())
    return MaybeLocal<Object>();
```

Reply:
Ah, thanks. I knew it wasn't perfect but couldn't figure out what the best way was. Thanks for the help